src/main/webapp/resources/js/components/samples/components/SampleAnalyses.jsx

```diff
@@ -1,43 +1,133 @@
+      (analysesList) => {
+        dispatch(setSampleAnalyses({ analyses: analysesList }));
+      }
+    );
+  }, []);
+
   // Columns for the table
   const columns = [
     {
-      title: i18n("SampleAnalyses.sampleId"),
-      key: "sample_id",
+      title: i18n("SampleAnalyses.analysisSubmissionName"),
+      dataIndex: "name",
+      key: "name",
+      ellipsis: true,
+      width: 250,
+      render(name, data) {
+        return (
+          <a
+            className="t-analysis-name"
+            href={setBaseUrl(`analysis/${data.id}`)}
+            title={name}
+            target="_blank"
+          >
+            {name}
+          </a>
+        );
+      },
     },
     {
-      title: i18n("SampleAnalyses.sampleName"),
-      key: "sample_name",
+      title: i18n("SampleAnalyses.analysisType"),
+      dataIndex: "analysisType",
+      key: "analysisType",
+      width: 150,
     },
     {
-      title: i18n("SampleAnalyses.analysisSubmission"),
-      key: "analysis_name",
+      title: i18n("SampleAnalyses.createdDate"),
+      dataIndex: "createdDate",
+      key: "createdDate",
+      width: 200,
+      render: (date) => (
+        <span className="t-analysis-created-date">
+          {date ? formatInternationalizedDateTime(date) : ""}
+        </span>
+      ),
     },
     {
-      title: i18n("SampleAnalyses.analysisType"),
-      key: "analysis_type",
+      title: i18n("SampleAnalyses.analysisState"),
+      dataIndex: "state",
+      key: "analysisType",
+      width: 100,
+      render: (state) => <SampleAnalysesState state={state} />,
     },
   ];
 
+  /*
+  Filter displayed submissions by name or type
+   */
+  const searchSubmissions = (searchStr) => {
+    if (
+      searchStr.trim() === "" ||
+      searchStr === "undefined" ||
+      searchStr === null
+    ) {
+      setFilteredSubmissions(analyses);
+    } else {
+      searchStr = String(searchStr).toLowerCase();
+      const submissionsContainingSearchValue = analyses.filter(
+        (submission) =>
+          submission.name.toLowerCase().includes(searchStr) ||
+          submission.analysisType.toLowerCase().includes(searchStr)
+      );
+
+      setFilteredSubmissions(submissionsContainingSearchValue);
+    }
+  };
+
   return (
-    <Space style={{ display: "block" }} direction="vertical">
-      <Search
-        placeholder={i18n("SampleAnalyses.inputSearchText")}
-        onChange={(e) => console.log(e.target.value)}
-        allowClear={true}
-        className="t-sample-search-input"
-      />
-      <Table bordered columns={columns} className="t-sample-analyses" />
-    </Space>
+    <Row gutter={[16, 16]}>
+      <Col span={24}>
+        <Search
+          placeholder={i18n("SampleAnalyses.inputSearchText")}
+          onChange={(e) => searchSubmissions(e.target.value)}
+          allowClear={true}
+          className="t-sample-analyses-search-input"
+        />
+      </Col>
+      <Col span={24}>
+        <Table
+          bordered
```

Comment:
Try removing this and see how it looks.  I don't think we render a lot of bordered tables, but if it adds somethine you feel is important let me know and we can keep it.